process.go

```diff
@@ -180,6 +187,10 @@ func (p *process) Run(ctx context.Context) error {
 		return p.eventLoop(ctx)
 	})
 
+	group.Go(func() error {
```

Comment:
I think csv writing needs to be an off-by-default option? Otherwise unaware users (umbrel?) slowly see their disk fill up.